setup.cfg

```diff
@@ -84,3 +84,11 @@ console_scripts =
 
 [options.packages.find]
 where = lib
+
+[mypy_config]
+color_output = True
+error_summary = True
+
+; ignore_missing_imports=False
+; disallow_untyped_calls=True
+; warn_redundant_casts=True
```

Comment:
This must go to a mypy config. I intentionally keep configs of different tools separate.